src/client/content_mapblock.cpp

```diff
@@ -998,6 +1026,22 @@ void MapblockMeshGenerator::drawPlantlike()
 		break;
 	}
 
+	if (is_rooted) {
```

Comment:
If wallmounted is also supposed to work for plantlike alone why do this?

Reply:
It seems to work so it can't be incorrect.